docs/syntax/c4.md

```
@@ -210,6 +210,26 @@ UpdateRelStyle(customerA, bankA, $offsetY="60")
+
+```yaml
+---
+config:
+  c4:
+    showStereotypes: false
+---
+```
+
+A single element can override the diagram-wide setting with `$showStereotype`, in either direction:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)